### PR TITLE
As a user, I can click an attribution class to learn more about our classification system

### PR DESCRIPTION
**Pull request by lilyfromseattle**
__User Story__
As a user, I can tap on the attribution class below the tombstone to learn more. Tapping on the link will open a new page with all of Artsy’s classifications.

I can click the back button or “OK” to go back to the artwork page.

__What I Did__
I added a view controller for the attribution class FAQ and updated ARRootViewController. I added dismissNavigationViewController to ARSwitchboardModule and Switchboard.tsx because they previously only had dismissModalViewController, and the attribution class FAQ is not a modal. I updated AppDelegate.m to include jumpToArtworkAttributionClassFAQ.

__Spec__
https://zpl.io/aX3O198

Screen Shot 2019-06-24 at 10 08 37 AM
Screen Shot 2019-06-24 at 10 08 46 AM

#skip_new_tests

### CODE REVIEW

File: Pod/Classes/ViewControllers/ARComponentViewController.m [hunk 1]
```diff
@@ -87,6 +87,12 @@ - (void)viewWillAppear:(BOOL)animated
     NSMutableDictionary *appProperties = [self.rootView.appProperties mutableCopy];
     appProperties[@"isVisible"] = @YES;
     self.rootView.appProperties = appProperties;
+    
+    double delayInSeconds = 5.0;
```
Comment (sweir27) [inline Pod/Classes/ViewControllers/ARComponentViewController.m]: Curious where this 5.0 came from? (Was it spec'ed out, is it standard, etc.?)

File: Example/Emission.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -654,9 +651,6 @@
 				"${TARGET_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}/AGaramondPro-Regular.otf",
 				"${TARGET_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}/AGaramondPro-Semibold.otf",
 				"${TARGET_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}/AVG65lig.otf",
-				"${TARGET_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}/ReactNativeAGaramondPro-Italic.otf",
```
Comment (sweir27) [inline Example/Emission.xcodeproj/project.pbxproj]: cc @alloy do these changes look 👍 or 👎 ?

File: src/lib/Scenes/Artwork/Components/__tests__/ArtworkTombstone-tests.tsx [hunk 3]
```diff
@@ -35,6 +35,21 @@ describe("ArtworkTombstone", () => {
     expect(SwitchBoard.presentNavigationViewController).toHaveBeenCalledWith(expect.anything(), "/artist/andy-warhol")
   })
 
+  it("redirects to attribution class faq page when attribution class is clicked", () => {
+    const component = mount(
+      <Theme>
+        <ArtworkTombstone artwork={artworkTombstoneArtwork as any} />
```
Comment (sweir27) [inline src/lib/Scenes/Artwork/Components/__tests__/ArtworkTombstone-tests.tsx]: I'm not sure what error was being thrown, but is it possible to type this? (i.e. what happens if you get rid of the `as any`)

File: Pod/Classes/ViewControllers/ARComponentViewController.m [hunk 4]
```diff
@@ -87,6 +87,12 @@ - (void)viewWillAppear:(BOOL)animated
...
+    dispatch_time_t popTime = dispatch_time(DISPATCH_TIME_NOW, delayInSeconds * NSEC_PER_SEC);
+    dispatch_after(popTime, dispatch_get_main_queue(), ^(void){
+        [self dismissViewControllerAnimated:YES completion:nil];
+    });
```
Comment (alloy) [inline Pod/Classes/ViewControllers/ARComponentViewController.m]: Was this for testing?

File: Example/Emission.xcodeproj/project.pbxproj [hunk 5]
```diff
@@ -654,9 +651,6 @@
...
-				"${TARGET_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}/ReactNativeAGaramondPro-Regular.otf",
-				"${TARGET_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}/ReactNativeAGaramondPro-Semibold.otf",
```
Comment (alloy) [inline Example/Emission.xcodeproj/project.pbxproj]: Looks like you have an outdated Pods installation, this change shouldn’t happen. Can you try `rm -rf Pods/Artsy+UIFonts && bundle exec pod install` and then re-open the workspace in Xcode.app?

File: Pod/Classes/ViewControllers/ARArtworkAttributionClassFAQViewController.m [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+                        moduleName:@"ArtworkAttributionClassFAQ"
+                 initialProperties:@{}];
+}
+
+- (void)viewDidLoad;
+{
+    [super viewDidLoad];
+    
+
+}
```
Comment (alloy) [inline Pod/Classes/ViewControllers/ARArtworkAttributionClassFAQViewController.m]: This empty override can be removed if nothing will be added here.

File: Pod/Classes/ViewControllers/ARComponentViewController.m (hunk 4 above)
Comment (lilyfromseattle) [reply Pod/Classes/ViewControllers/ARComponentViewController.m]: Yup my bad!

File: Pod/Classes/ViewControllers/ARComponentViewController.m (hunk 1 above)
Comment (lilyfromseattle) [reply Pod/Classes/ViewControllers/ARComponentViewController.m]: Oh that was a testing thing I'll remove it

File: Example/Emission.xcodeproj/project.pbxproj (hunk 5 above)
Comment (lilyfromseattle) [reply Example/Emission.xcodeproj/project.pbxproj]: Yup I think that worked!

**Pull request merged**